fuzz/cmp.c

```diff
@@ -176,13 +176,26 @@ int FuzzerTestOneInput(const uint8_t *buf, size_t len)
         return 0;
 
     in = BIO_new(BIO_s_mem());
-    OPENSSL_assert((size_t)BIO_write(in, buf, len) == len);
+    if ((size_t)BIO_write(in, buf, len) != len) {
```

Comment:
Hmm. I am unconvinced by the removal of the assert here. This really shouldn't fail and if it does we want to know about it - it could indicate some significant problem - which is kind of the point of a fuzzer.

Reply:
Hmm, but if we eventually want to fuzz with reproducible allocation failures this assert would trigger.